helium-dev/src/main/java/org/apache/zeppelin/helium/ZeppelinDevServer.java

```diff
@@ -39,7 +39,7 @@
   private DevInterpreter interpreter = null;
   private InterpreterOutput out;
   public ZeppelinDevServer(int port) throws TException, IOException {
-    super(null, port);
+    super(null, port, "");
```

Comment:
maybe default to ":" like in other cases?

Reply:
fixed